packages/common/src/store/account/sagas.ts

```diff
@@ -86,6 +87,16 @@ export function* fetchAccountAsync({ isSignUp = false }): SagaIterator {
     },
     true // force refresh to get updated user w handle
   )
+  const { data: accountCidData } = yield* call(
+    [sdk.full.cidData, sdk.full.cidData.getMetadata],
+    {
+      metadataId: accountData?.user?.metadata_multihash
+    }
+  )
+  accountData.user = {
```

Comment:
oof extra network request not ideal, but I get it.
Is cid metadata the only way to get associated wallets? How was this working before with the rewards page?

Reply:
currently yes. there's a path to making this just on the account endpoint, as with collectibles but for now this is the same behavior as we had before